census/controllers/census.js

```diff
@@ -219,6 +221,39 @@ var submit = function(req, res) {
   });
 };
 
+let _getDiscussionURL = function(req, dataset, place) {
+  /*
+    If `submission_discussion_url` is defined in settings and it is in the
+    format: https://discuss.okfn.org/c/<topic>/<subtopic>, return a new topic
+    url with a prepopulated topic for place and dataset. Otherwise, return the
+    original `submission_discussion_url` without modification. If
+    `submission_discussion_url` is undefined return an empty string.
+  */
+  let submissionDiscussionURL =
+    _.get(req.params.site.settings, 'submission_discussion_url', '');
+  let parsedURL = url.parse(submissionDiscussionURL);
+  // URL is a discourse link
+  if (parsedURL.hostname === 'discuss.okfn.org') {
```

Comment:
First thing here: we shouldn't hardcode stuff and affect platform reusability.

Second thing, it should be the user's responsibility to provide a valid Discourse URL (so maybe the env var should be renamed to `discourse_url`). Further support for other platforms or multiple Discourse URLs will be easy to add if needed. As I see it, this is a broad scope variable that is treated exclusively as one of the possible cases, even if we don't need that scalability now.

That would both simplify the code here and extract the hardcoded URL from the function.